tools/tune_fp8_cutlass.py

```
@@ -0,0 +1,496 @@
+  # Mode 3: Tune across multiple model families (shapes are merged/de-duped)
+  python tools/tune_fp8_cutlass.py \
+      --model_path /path/to/qwen72b /path/to/gpt-oss-20b /path/to/deepseek \
+      --tune --output csrc/core/kernel/cuda/gemm_lowp/gemm_fp8_blockwise_tuned.h
+"""
+import argparse
+import ctypes
+import glob
+import json
+import os
+import site
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)